frontend/src/pages/home/index.tsx

```diff
@@ -72,8 +73,10 @@ export default function Home() {
                         <p className="text-gray-700 text-sm">{baby.head_circumference}</p>
                     </div>
                     <button className="w-full inline-flex justify-center mt-1 rounded-md border bg-background px-2 py-2 text-sm font-medium shadow-sm cursor-pointer hover:text-blue-600 transition-all duration-300">
-                        <FiUserPlus className="h-4 w-4 mr-2" />
-                        Registrar Informações
+                        <Link href={`/registerbaby/${baby.id}`} className="flex items-center w-full justify-center">
```

Comment:
```suggestion
                        <Link href={`/registerbaby/${baby.name}`} className="flex items-center w-full justify-center">
```
Vamos exibir somente o nome do bebe.